src/explore-education-statistics-common/src/components/form/FormRadio.tsx

```diff
@@ -61,6 +67,15 @@ const FormRadio = ({
           {conditional}
         </div>
       )}
+
+      {children && (
+        <div
+          className="govuk-radios__conditional govuk-radios__conditional--hidden"
```

Comment:
A `conditional` prop has already been added to `FormRadio`. I think adding `id={`${id}-conditional`} is still probably worthwhile though.

I think I probably made a mistake by adding the `hidden` attribute to it as well, so that can probably be removed.

Reply:
It was added like this as `conditional` didn't work without the id... I will change the code and see if it works with the id.

Reply:
It was also to make it easier for Marv to just add HTML as HTML.